Stack.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stack
+{   public class Stack
+    {   List<object> listOfData;
+        Stack(List<object> list)
+        {
+            this.listOfData = list;
+        }
+        public  void PushData(object value)
+        {
+            listOfData.Add(value);
+        }
+        public object PopData()
+        {   try
+            {
+                object data = listOfData.ElementAt(listOfData.Count - 1);
+                listOfData.RemoveAt(listOfData.Count - 1);
+                return data;
+            }
+            catch (Exception)
+            {
+                return "The List Is Empty ";
+           }
+        }
+
+        public  void PrintData()
+        {
+            try
+            {
+                List<object> data = listOfData.Select(s => s).ToList();
```

Comment:
@noor848 1. could you please clarify what you are trying to do here? I see you read the listOfData list and moving it to another list, why?
2. and do you think if we have a real stack will print the data from it as order you print it?